.changeset/feat-treeview-is-top-level-selectable.md

```diff
@@ -0,0 +1,6 @@
+---
+'react-magma-dom': minor
+---
+
+feat(TreeView): Add isTopLevelSelectable prop to TreeView component
+```
```

Comment:
The 3 ticks at the bottom of the file are what need to be removed, the quotes around `isTopLevelSelectable` are fine and good.